app/pages/History/TransactionDetails.view/TransactionDetails.view.tsx

```diff
@@ -121,12 +120,8 @@ const TransactionDetails: FC<TransactionDetailsViewProps> = ({
 
   const renderSenderOrReceiver = () => {
     let aliasOrAddress: string | ReactNode;
-    if (assignedAddressId || address) {
-      aliasOrAddress = contact ? (
-        contact.alias
-      ) : (
-        <ShortCode code={assignedAddressId || address || ''} />
-      );
+    if (address) {
+      aliasOrAddress = contact ? contact.alias : <ShortCode code={address || ''} />;
```

Comment:
address is already valid here because of the initial `if` statment
```suggestion
      aliasOrAddress = contact ? contact.alias : <ShortCode code={address} />;
```